Mathlib/CategoryTheory/Galois/Basic.lean

```diff
@@ -164,6 +164,12 @@ lemma initial_iff_fibre_empty (X : C) : Nonempty (IsInitial X) ↔ IsEmpty (F.ob
     infer_instance
   exact Concrete.initial_iff_empty_of_preserves_of_reflects (F.obj X)
 
+/-- An object is not initial if and only if its fibre is nonempty. -/
+lemma not_initial_iff_fibre_nonempty (X : C) : (IsInitial X → False) ↔ Nonempty (F.obj X) := by
```

Comment:
```suggestion
/-- An object is not initial if and only if its fiber is nonempty. -/
lemma not_initial_iff_fiber_nonempty (X : C) : (IsInitial X → False) ↔ Nonempty (F.obj X) := by
```
In mathlib, USA-English is favored against UK-English. Then, I am afraid you should also rename `FibreFunctor` as `FiberFunctor`, etc.

Reply:
This is unfortunate, `FiberFunctor` looks horrible to my eyes, well I will of course try to adhere to the convention. Since I don't want to produce any merge conflicts with this and the parallel PR: Should I fix the spelling when these two are merged? Since I need to change this at quite a few places.

Reply:
The change for `FibreFunctor` can be done separately, but I would suggest that for the new definitions here, the change "fibre" -> "fibre" is applied immediately.